source/acquaman/SGM/SGMXASScanActionController.cpp

```diff
@@ -207,23 +207,23 @@ QString SGMXASScanActionController::buildNotes()
 }
 
 AMAction3* SGMXASScanActionController::createInitializationActions(){
-	AMControlMoveActionInfo3 *moveActionInfo;
-	AMControlMoveAction3 *moveAction;
-	AMControl *tmpControl;
+//	AMControlMoveActionInfo3 *moveActionInfo;
+//	AMControlMoveAction3 *moveAction;
+//	AMControl *tmpControl;
 
 	AMListAction3 *initializationActions = new AMListAction3(new AMListActionInfo3("SGM XAS Initialization Actions", "SGM XAS Initialization Actions"));
 
 
 	AMListAction3 *initializationStage1 = new AMListAction3(new AMListActionInfo3("SGM XAS Initialization Stage 1", "SGM XAS Initialization Stage 1"), AMListAction3::Parallel);
 
-    tmpControl = SGMBeamline::sgm()->grating();
-    double gratingSetPoint = configuration_->fluxResolutionGroup().controlNamed(tmpControl->name()).value();
-    AMAction3 *gratingAction = AMActionSupport::buildControlMoveAction(tmpControl->grating(), gratingSetPoint);
+//    tmpControl = SGMBeamline::sgm()->grating();
+//    double gratingSetPoint = configuration_->fluxResolutionGroup().controlNamed(tmpControl->name()).value();
+    AMAction3 *gratingAction = AMActionSupport::buildControlMoveAction(SGMBeamline::sgm()->grating(), configuration_->fluxResolutionGroup().controlNamed(SGMBeamline::sgm()->grating()->name()).value());
```

Comment:
That's my bad. I was fixing this up for building. I'll get them sorted out on my end first.